.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+on: push
+jobs:
+  build:
+    runs-on: macos-latest
+    env:
+      LDFLAGS: -L/usr/local/opt/openblas/lib
+      CPPFLAGS: -I/usr/local/opt/openblas/include
```

Comment:
you can set MAKEFLAGS onc in env instead of calling sysctl in each command to set the number of jobs

Reply:
maybe then repush to retrigger as I've just enabled actions in PRs from forks in openturns settings

Reply:
indeed

Reply:
or maybe the yml must already be present, like what we had for the sphinx artifacts